tests/typescript/conformance/internalModules/importDeclarations/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,382 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`circularImportAlias.ts 1`] = `
+// expected no error
+
+module B {
+    export import a = A;
+    export class D extends a.C {
+        id: number;
+    }
+}
+
+module A {
+    export class C { name: string }
+    export import b = B;
+}
+
+var c: { name: string };
+var c = new B.a.C();
+
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+// expected no error
+
+module B {
+  export import a = A;
+  export class D extends a.C {
+    id: number;
+  }
+}
+
+module A {
+
+  export class C {
+    name: string;
+  }export import b = B;
```

Comment:
The solution is likely to handle `\n` inside of the code that prints `module` instead of adding random hardlines before each of them.